app/scripts/lib/ens-ipfs/setup.js

```diff
@@ -25,9 +24,7 @@ export default function setupEnsIpfsResolver ({ provider, getCurrentNetwork, get
     if (tabId === -1 || getCurrentNetwork() !== '1') {
       return
     }
-    // parse ens name
-    const urlData = urlUtil.parse(url)
-    const { hostname: name, path, search, hash: fragment } = urlData
+    const { hostname: name, path, search, hash: fragment } = new URL(url)
```

Comment:
The WHATWG `URL` object doesn't contain a `path` property ([see this comparison](https://nodejs.org/api/url.html#url_url)). This `path` property from the deprecated API is equivalent to `${pathname}{search}`.

It looks like `pathname` is what was originally intended here. In `attemptResolve` below it is concatenated with `search`. This implies that if `search` was non-empty, we would see it duplicated on these resolved URLs :thinking: 

I will test this theory, and fix this in a separate PR if so.

Reply:
I have confirmed that this is a bug. It is addressed in #9674